src/components/Tx/SendTx/CurrencySelect.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Identicon from '../../Identicon'
+
+export default class CurrencySelect extends Component {
+  static displayName = 'CurrencySelect'
+
+  static propTypes = {
+    etherWallet: PropTypes.object,
```

Comment:
I would define this as PropTypes.shape() to show what the object should look like

e.g. from https://reactjs.org/docs/typechecking-with-proptypes.html:
```
 // An object taking on a particular shape
  optionalObjectWithShape: PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  }),
```